log/base_handler.ts

```diff
@@ -8,19 +8,25 @@ import {
 } from "./levels.ts";
 import type { LogRecord } from "./logger.ts";
 
+/** The formatter function */
 export type FormatterFunction = (logRecord: LogRecord) => string;
 const DEFAULT_FORMATTER: FormatterFunction = ({ levelName, msg }) =>
   `${levelName} ${msg}`;
 
+/** The options of {@linkcode BaseHandler} */
 export interface BaseHandlerOptions {
+  /** The log formatters */
   formatter?: FormatterFunction;
 }
 
+/** The base class of handlers */
 export abstract class BaseHandler {
   #levelName: LevelName;
   #level: LogLevel;
+  /** The log formatters */
   formatter: FormatterFunction;
 
+  /** Construct a {@linkcode BaseHandler} */
```

Comment:
Ditto for the other constructors
```suggestion
  /** Constructs a new instance. */
```